Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM centos:centos7
 
 ADD . /usr/src/sriov-network-device-plugin
 
+ENV HTTP_PROXY $http_proxy
```

Comment:
These ENV variables will persist in the build/final images, I'm not sure if it'll cause any problem when the image is running with HTTP(S)_PROXY set as None, any thought on that?

Reply:
Sounds good, Thanks